src/App.xaml.cs

```diff
@@ -126,7 +126,7 @@ public App()
             services.AddSingleton<IScreenReaderService, ScreenReaderService>();
             services.AddSingleton<IComputeSystemService, ComputeSystemService>();
             services.AddSingleton<IComputeSystemManager, ComputeSystemManager>();
-            services.AddSingleton<ToastNotificationService>();
+            services.AddTransient<NotificationService>();

```

Comment:
Switching from Singleton to Transient is a significant change. Do we really want it to be Transient?

Reply:
Yes, want this to be reusable.